packages/edit-site/src/components/page-pages/index.js

```diff
@@ -328,18 +338,30 @@ export default function PagePages() {
 						? 'edit-site-page-pages-list-view'
 						: null
 				}
-				title={ __( 'Pages' ) }
+				title={
+					showDetails
+						? decodeEntities(
+								pageRecord?.title?.rendered ||
+									__( '(no title)' )
+						  )
+						: __( 'Pages' )
+				}
+				onClose={ showDetails ? onDetailsChange : null }
 			>
-				<DataViews
-					paginationInfo={ paginationInfo }
-					fields={ fields }
-					actions={ actions }
-					data={ pages || EMPTY_ARRAY }
-					isLoading={ isLoadingPages || isLoadingAuthors }
-					view={ view }
-					onChangeView={ onChangeView }
-					onSelectionChange={ onSelectionChange }
-				/>
+				{ showDetails && <Details item={ pageRecord } /> }
```

Comment:
I wish I could reuse the existing "details" components instead of having to create `Details`, but the current ones do too many things (assume they are rendered in the sidebar, deal with router logic, fetch data, etc.). We should revisit when making this stable.